script/release-npm

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+echo "####################################################"
+echo "NPM version:"
+echo "####################################################"
+
+read -p "MAJOR: " major
+read -p "MINOR: " minor
+read -p "PATCH: " patch
+
+# Update package.json version
+sed -E -i.bak "s/\"version\": \"(.*)?\"/\"version\": \"$major.$minor.$patch\"/g" package.json
```

Comment:
This looks quite fragile. Could we look at shelling out to Ruby or perhaps using something like `jq` here?